### DISCUSSION TITLE
阿里云的直链是不是有时间限制啊，过期了会失效，好像过了一天就失效了

### DISCUSSION THREAD

**Discussion by dary123** [AlistGo/alist#812] [Q&A]
阿里云的直链是不是有时间限制啊，过期了会失效，好像过了一天就失效了

**Comment by xhofe** [comment]
- 永久有效：https://alist.xhofe.top/d/%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98/%E5%9B%BE%E7%89%87/android-github-actions-setup-image.png?sign=0a504cb04b977dbb
- 4小时：https://bj29.cn-beijing.data.alicloudccp.com/5fe4259901e1c339435646a5a63cc52be58871c0%2F5fe42599b32c647c1cda4f6ab4edbfb04a1c24fb?di=bj29&dr=541838450&f=61dddf5e98c7b00a831b450dab0c5b0b899a33c8&response-content-disposition=attachment%3B%20filename%2A%3DUTF-8%27%27Ya7b0e.png&u=b0caff13926144c7b827e51f50150cff&x-oss-access-key-id=LTAIsE5mAn2F493Q&x-oss-expires=1648304327&x-oss-signature=tX0EasOtmfyJPF%2Fxeo90HUjOety5tP8pknkKEOJSSdo%3D&x-oss-signature-version=OSS2

**Reply by dary123** [reply]
那到底是直链不会失效，还是直链(decode)不会失效

**Reply by xhofe** [reply]
都不会失效。
